README.md

```diff
@@ -343,7 +343,7 @@ I am just a college student and this is one of my amateur projects(I need to fin
 
 ## Contributing
 
-Annie is an open source project and built on the top of open source projects. If you are interested, you are welcome to contribute, let's make Annie better, together 💪
```

Comment:
Is there any grammatical error in this sentence? If not, I prefer not to change it.